app/views/miq_request/_prov_field.html.haml

```diff
@@ -298,13 +298,7 @@
             - check = @miq_request.id
           - else
             - check = 'new'
-          = render(:partial => "layouts/tree",
-                   :locals => {:tree_id       => "all_tags_treebox",
-                               :tree_name     => "all_tags_tree",
-                               :bs_tree       => @all_tags_tree,
-                               :oncheck       => "miqOnCheckProvTags",
```

Comment:
also remove the `miqOnCheckProvTags` function please (and from the whitelist)?

(If it was used `miq_prov_vars` would still need the split .. but it is dead now :).)